From 80fba698ff47dd122a8fbb083c8433cba5933e84 Mon Sep 17 00:00:00 2001
From: Jason Furmanek <furmanek@us.ibm.com>
Date: Wed, 28 Jul 2021 13:19:30 -0400
Subject: [PATCH] adjust protobuf github repo

---
 WORKSPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WORKSPACE b/WORKSPACE
index 495ed63..c9ba72e 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -22,7 +22,7 @@ git_repository(
     name = "com_google_protobuf",
     # v3.8.0
     commit = "09745575a923640154bcf307fba8aedff47f240a",
-    remote = "https://github.com/google/protobuf.git",
+    remote = "https://github.com/protocolbuffers/protobuf.git",
 )
 
 # Required by protobuf 3.8.0.
-- 
2.23.0

