From 1b064e089ff3dda10807caf89d25d9f594ab9959 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Tue, 29 Mar 2022 03:50:50 -0400
Subject: [PATCH] Updated zlib URL

---
 WORKSPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WORKSPACE b/WORKSPACE
index 495ed63..c2656e2 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -31,7 +31,7 @@ http_archive(
     build_file = "@com_google_protobuf//:third_party/zlib.BUILD",
     sha256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1",
     strip_prefix = "zlib-1.2.11",
-    urls = ["https://zlib.net/zlib-1.2.11.tar.gz"],
+    urls = ["https://zlib.net/zlib-1.2.11.tar.gz", "https://www.zlib.net/fossils/zlib-1.2.11.tar.gz"],
 )
 
 # Required by protobuf 3.8.0.
-- 
2.32.0

